deepmd/utils/network.py

```diff
@@ -44,6 +49,12 @@ def one_layer(inputs,
                             b_initializer, 
                             trainable = trainable)
         variable_summaries(b, 'bias')
+
+        if mixed_prec is not None and outputs_size != 1:
```

Comment:
I do not like this idea. 
For dipole and polar, the size of output layer is not 1, but they are using fp16, which is not what we want.